apiary.apib

```diff
@@ -95,16 +123,51 @@ This endpoint allows asset import data to be found.
 + Response 200 (application/json; charset=utf-8)
     + Attributes (Asset Import)
 
-## Asset Export [/accounts/{accountId}/export/assets]
+## Asset Import Log Collection [/accounts/{accountId}/import/assets/{assetImportId}/logs{?page,pagesize,orderby}]
```

Comment:
Do we need to be able to filter the logs by logType? (to filter and show only errors)